acceptance/bundle/scripts/databricks.yml

```diff
@@ -0,0 +1,11 @@
+bundle:
+  name: scripts
+
+experimental:
+  scripts:
+    preinit: "python3 ./myscript.py $EXITCODE preinit"
+    postinit: "python3 ./myscript.py 0 postinit"
+    prebuild: "python3 ./myscript.py 0 prebuild"
+    postbuild: "python3 ./myscript.py 0 postbuild"
+    predeploy: "python3 ./myscript.py 0 predeploy"
+    postdeploy: "python3 ./myscript.py 0 postdeploy"
```

Comment:
We don't really test this one, correct? Should we have an integration test instead which does successful deploy so we can cover this as well?

Reply:
It is useful to have it there to see that it was not run.